packages/x-components/src/adapter/util.ts

```diff
@@ -15,7 +15,7 @@ export function configureAdapterWithToysrus(
 ): EmpathyAdapterBuilder {
   adapterBuilder
     .addMapper((_, result: Result) => {
-      result.url = `./product_page.html?productId=${result.id}`;
+      result.url = `${location.origin}/product_page.html?productId=${result.id}&test=t e s t`;
```

Comment:
Currently we are only able to track the PDP add to cart if we are using complete urls. We should think how to handle the case of relative urls.

Reply:
can compare only the path of the URL?